e2e_test/ddl/show.slt

```diff
@@ -7,15 +7,21 @@ create materialized view mv3 as select sum(v1) as sum_v1 from t3;
 statement ok
 create view v3 as select sum(v2) as sum_v2 from t3;
 
-query TTT
+statement ok
+comment on column t3.v2 is 'Lorem ipsum dolor sit amet';
```

Comment:
Please add some tests for `comment on table xxxxx`.

Reply:
Currently `describe` query does not return table description. 
Should I select directly from `rw_descripton`/`pg_description` for testing?

btw, should we put table description into the result of describe query?

Reply:
Since we have `primary key` row and so, I guess it's reasonable to also add a row with first field set to "description"? Not sure, @yezizp2012

Reply:
Yes, sure you can use `rw_descripton`/`pg_description` for testing. And for `describe` I think it's acceptable for me to add table description as well.